spec/models/full_contribution_calculator_spec.rb

```diff
@@ -15,8 +15,20 @@
   end
 
   describe '#employee_percent' do
-    it 'returns 1' do
-      expect(subject.employee_percent).to eq(1)
+    context 'yearly salary less than Lower Earnings Threshold' do
+      let(:salary_per_year) { 5_000 }
+
+      it 'returns 0' do
```

Comment:
@margOnline Is there any cucumber features that cover this feature in the past? Is it worth adding one?

Reply:
No cuke features that specifically cover this feature but the framework is in place and I agree that one should be created.

Reply:
@scott-ad-riley wdyt?

Reply:
Sorry - thought I'd clicked the button, I'll add the feature spec.

Reply:
Cool! I will change to WIP the PR title then.